src/App.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import './App.css';
+import resumeJson from './assets/resume.json';
+
+import Experience from "./components/Experience/Experience";
+import Study from "./components/Study/Study";
+import Award from "./components/Award/Award";
+import Skill from "./components/Skill/Skill";
+import Language from "./components/Language/Language";
+import Interest from "./components/Interest/Interest";
+import Reference from "./components/Reference/Reference";
+
+function App() {
+  const resumeData = resumeJson;
```

Comment:
I wonder if we could have capitalized that resumeJson was already a well built JSON object and further broke it down, referencing variables that are assumed to be in the object such as:

const { basics } = resumeJson.

The above may allow less complex referencing, not that the resume JSON was overly complex.